integration/backward_compatibility_test.go

```diff
@@ -73,7 +75,7 @@ func runBackwardCompatibilityTestWithChunksStorage(t *testing.T, previousImage s
 
 	// Start other Cortex components (ingester running on previous version).
 	ingester1 := e2ecortex.NewIngester("ingester-1", consul.NetworkHTTPEndpoint(), flagsForOldImage, previousImage)
-	distributor := e2ecortex.NewDistributor("distributor", consul.NetworkHTTPEndpoint(), flagsForOldImage, "")
+	distributor := e2ecortex.NewDistributor("distributor", consul.NetworkHTTPEndpoint(), ChunksStorageFlags, "")
```

Comment:
Why this change? This change the meaning of the test. We're not testing anymore an old distributor anymore.

Reply:
Sorry. I misread it. We're actually using the new image, so the previous code looks incorrect. Thanks for fixing!